pkg/replication/replication_mode.go

```diff
@@ -489,7 +489,7 @@ func (m *ModeManager) tickUpdateState() {
 			m.drSwitchToSyncRecover()
 			break
 		}
-		if !reflect.DeepEqual(m.drGetAvailableStores(), stores[primaryUp]) && m.drCheckStoreStateUpdated(storeIDs(stores[primaryUp])) {
+		if !reflect.DeepEqual(m.drGetAvailableStores(), storeIDs(stores[primaryUp])) && m.drCheckStoreStateUpdated(storeIDs(stores[primaryUp])) {
```

Comment:
How about `checkStoreStatus` returns `[][]*core.StoreInfo` and `[][]uint64`

Reply:
BTW, it may be more readable ` primaryUpStores := storeIDs(stores[primaryUp]` https://github.com/lhy1024/pd/blob/2452a4de9440d2c5d5c098c0555a003ee928260a/pkg/replication/replication_mode.go#L469